test/Dialect/FSM/print-graph.mlir

```diff
@@ -0,0 +1,42 @@
+// RUN: circt-opt --fsm-print-graph %s -o %t 2>&1 | FileCheck %s
+
+// CHECK: [[IDLE:.*]] [shape=record,label="{IDLE|fsm.output %true : i1}"];
+// CHECK: [[IDLE]] -> [[BUSY:.*]][label="fsm.return %arg0"];
+// CHECK: [[BUSY]] [shape=record,label="{BUSY|fsm.output %false : i1}"];
+// CHECK: [[BUSY]] -> [[BUSY]][label="%0 = arith.cmpi ne, %cnt, %c0_i16 : i16\nfsm.return %0"];
+// FIX ME: FileCheck doesn't want to play nice with the following line (intended check: "[[BUSY]] -> [[IDLE]][label="%0 = arith.cmpi eq, %cnt, %c0_i16 : i16\nfsm.return %0"];")
```

Comment:
Can you use CHECK{LITERAL} to help with this?

https://llvm.org/docs/CommandGuide/FileCheck.html#directive-modifiers

Reply:
I don't think that would apply - the `BUSY` and `IDLE` variables capture names generated from the runtime addresses of the underlying nodes (default DOT graph traits behaviour) and thus change on every run with the same input.